src/utils/application.data.ts

```diff
@@ -89,7 +89,7 @@ export const checkMOsDetailsEntered = (appData: ApplicationData): boolean => {
   return Boolean( appData[ManagingOfficerKey]?.length || appData[ManagingOfficerCorporateKey]?.length ) ;
 };
 
-export const hasAddedOrCeasedABO = (appData: ApplicationData): boolean => {
+export const hasAddedOrCeasedBO = (appData: ApplicationData): boolean => {
   return allBeneficialOwners(appData).some((bo) => (bo.ceased_date && Object.keys(bo.ceased_date).length !== 0) || !bo.ch_reference);
```

Comment:
some(bo => (bo.ceased_date && Object.keys(bo.ceased_date).length !== 0)) || !bo.ch_reference);
the brackets arround && is not needed but I think it's clearer with them in.